packages/components/tests/dummy/app/templates/components/side-nav.hbs

```diff
@@ -616,12 +624,16 @@
     </SG.Item>
     <SG.Item @label="Icon + text + yielded content">
       <Hds::SideNav::List::Link @icon="boundary" @text="Boundary">
-        <Shw::Placeholder @height="20px" @text="yielded content" />
+        {{! template-lint-disable no-inline-styles }}
+        <Shw::Placeholder style="flex-grow: 1; height: 20px; width: auto;" @text="yielded content" />
```

Comment:
@didoo Would it be better to add a classname to these placeholders and move the inline styles there?

(I had to override the default `width: 100%` & add `flex-grow: 1` so the placeholder would be more flexible and not force "Boundary" text to wrap. Adding the inline stye overrode styles from the args so I had to use all inline styles vs. `height` & `width` args.)

Reply:
I would prefer to introduce a `@flex` argument so that you can say `@flex="1 1 0" and this _should_ fix the problem). See how the `@background` is done: https://github.com/hashicorp/design-system/blob/main/website/app/components/doc/placeholder/index.js#L56-L58

Reply:
BTW, to add inline styles (in the showcase) there is a special helper `{{style ...}}`

Reply:
Added new flex argument & updated in Showcase.